src/localStorage.js

```diff
@@ -0,0 +1,11 @@
+export function setSortingIndex(todoList) {
+  return todoList.map((item, index) => { item.index = index; return item; });
+}
+
+export function storeList(todoList) {
+  localStorage.setItem('todo-list', JSON.stringify(todoList));
+}
+
+export function getList() {
+  return JSON.parse(localStorage.getItem('todo-list'));
```

Comment:
- [ ] Here as well. I would recommend using the proper ES6 syntax to fulfill the project requirement and also writing clean code as ES6 syntax covers more work in less code.